README.md

```diff
@@ -293,3 +293,7 @@ const unleash = new UnleashClient({
 **NOTES: ⚠️**
 If `bootstrapOverride` is `true` (by default), any local cached data will be overridden with the bootstrap specified.   
 If `bootstrapOverride` is `false` any local cached data will not be overridden unless the local cache is empty.
+
+## Manage your own refresh mechanism
+
+If you do not want to use the internal refresh mechanism provided by the `refreshInterval` or `metricsInterval` options, you can handle it yourself using the public `updateToggles` and `sendMetrics` methods.
```

Comment:
e.g. "You can opt out of the Unleash feature flag auto-refresh mechanism by not calling the start method. In this case, it becomes your responsibility to call updateToggles and sendMetrics. This approach is useful in environments that do not support the setInterval API, such as service workers."

Reply:
We need to call the `start` method (no initial fetch otherwise) but what is needed is to set `disableMetrics: true` and/or `disableRefresh: true` in the options in order to handle it ourselves.

Reply:
Ah of course. You're right. Let's document it then :)

Reply:
Doc updated